src/search.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * @copyright Copyright (c) 2024 Fon E. Noel NFEBE <me@nfebe.com>
+ *
+ * @author Vincent Petry <me@nfebe.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import Vue from 'vue'
+
+import { getRequestToken } from '@nextcloud/auth'
+import { emit } from '@nextcloud/event-bus'
+import { translate, translatePlural } from '@nextcloud/l10n'
+import { generateFilePath } from '@nextcloud/router'
+
+import RoomSelector from './components/RoomSelector.vue'
+
+import { fetchConversation } from './services/conversationsService.js'
+
+import '@nextcloud/dialogs/style.css'
+
+(function(OC, OCA, t, n) {
+
+	/**
+	 *
+	 */
+	function init() {
+		if (!OCA.Core) {
+			return
+		}
+		console.debug('Initializing unified search plugin-filters from talk')
+		OCA.Core.UnifiedSearch.registerFilterAction({
+			id: 'talk-message',
+			label: t('spreed', 'In Conversation'),
+			icon: '/apps/spreed/img/app.svg',
```

Comment:
Not sure that path is picked correctly. Might be because I have it in 'apps-extra'?
![image](https://github.com/nextcloud/spreed/assets/93392545/96d6a056-b107-42bc-9b6b-498736b6aea0)

Reply:
Might be better, but it returns the same path for me

Reply:
Maybe there is a better of obtaining the app icon from within the app? `imagePath()` looks like it would send a full URL, which would again be passed to `generateUrl` since the other providers from the API send their icons in this format.

Reply:
```suggestion
			icon: imagePath('spreed', 'app.svg'),
```

Now works correctly for me with that change: 
![image](https://github.com/nextcloud/spreed/assets/93392545/d32a0e11-e8c1-41db-94f1-05c6a23b8640)